cms/tiny_table_block/static/tiny_table_block/js/tiny-table-block.js

```diff
@@ -0,0 +1,74 @@
+/* global tinymce */
```

Comment:
Hopefully this vendor files doesn't live here long if we can open source and package it separately. Did you consider npm installing?

Reply:
I have, but I am using a subset (tinymce + 2 or 3 plugins). Also, bundling it means downstream doesn't need to worry about FE tooling